build.zig

```diff
@@ -127,36 +129,54 @@ pub fn build(b: *std.build.Builder) !void {
         .root_source_file = .{ .path = "src/config_gen/config_gen.zig" },
     });
 
-    const gen_cmd = b.addRunArtifact(gen_exe);
-    gen_cmd.addArgs(&.{
-        "--readme-path",
-        b.pathFromRoot("README.md"),
-        "--generate-config-path",
-        b.pathFromRoot("src/Config.zig"),
-        "--generate-schema-path",
-        b.pathFromRoot("schema.json"),
-    });
-    if (b.args) |args| gen_cmd.addArgs(args);
-
-    const gen_step = b.step("gen", "Regenerate config files");
-    gen_step.dependOn(&gen_cmd.step);
+    const gen_config_cmd = gen_config_cmd: {
```

Comment:
Removed,  I added this for more clearier patches (everything is visually scoped) if distro's package wants to hardcode it for some reason or similar, like:
```diff
 const gen_version_data_cmd = gen_version_data_cmd: {
     const cmd = b.addRunArtifact(gen_exe);
-    cmd.addArgs(&.{ "--generate-for-version", compiler_version });
-    if (override_langref_file_path) |override_path| {
-        cmd.addArg("--langref_path");
-        cmd.addFileArg(.{ .path = override_path });
-    }
-    cmd.addArg("--generate-version-data-path");
-    cmd.addFileArg(version_data_path);
-    if (b.args) |args| cmd.addArgs(args);
-    break :gen_version_data_cmd cmd;
+    cmd.addArgs(&.{
+        "--generate-for-version",
+        "master",
+        "--langref-path",
+        "hardcoded",
+        "--generate-version-data-patch",
+        "another_hardcoded",
+    });
 };
 ```
 
 And because it was more clearier for me, but all this is not very important, so I dropped it now.